data-processing/entities/terms_v1/test_terms.py

```diff
@@ -0,0 +1,26 @@
+def test_extract_sentences():
+    extractor = TermExtractor()
+    terms = list(
+        extractor.parse(
+            "main.tex",
+            "/title (towards interpretable semantic segmentation via gradient-weighted class activation mapping)",
+        )
+    )
+    assert len(terms) == 3
+    term1 = terms[0]
+    assert term1.term == "segmentation"
+    assert term1.start == 1666
+    assert term1.end == 1677
+    assert term1.context_str == "/title (towards interpretable semantic segmentation via gradient-weighted class activation mapping)"
```

Comment:
This should be `context_tex`

Reply:
Also, there need not be a test for `context_str`, as it's for debugging purposes rather than serving an important functional role within the pipeline